lib/validate/static_utils.rb

```diff
@@ -17,7 +22,11 @@ def lint_tf
 def style_tf
   # Do the style checking on current working folder.
   print "INFO: Styling Terraform configurations...\n".yellow  
-  message = `terraform fmt -check=true 2>&1`
+  if ENV['TERRAFORM_VERSION'].start_with?("0.12")
```

Comment:
Can we add a fallback to to `0.11` if `0.12` is not set ? If user runs the test in native environment rather than container, there might be no `TERRAFORM_VERSION` defined.

Reply:
Ok, will test for ```ENV['TERRAFORM_VERSION'].start_with?('0.12')```, falling back to the 0.11 syntax using the ```else``` clause.

Reply:
Committed in [0985c2c](0985c2c)